index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Farida's Logo</a>
+      <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
+    </header>
+    <section id="intro">
+      <h2>I'm Farida Glad to see you!</h2>
```

Comment:
- [ ] Kindly change the `h2` tag to the `h1` tag since this is the main heading of the page.